scib/metrics/clustering.py

```diff
@@ -6,27 +6,40 @@
 from .nmi import nmi
 
 
-def opt_louvain(adata, label_key, cluster_key, function=None, resolutions=None,
-                use_rep=None,
-                inplace=True, plot=False, force=True, verbose=True, **kwargs):
+def opt_louvain(
```

Comment:
Should this be in the preprocessing module rather than metrics?

Reply:
It's part of a few metrics, so I think prob not. Preprocessing is for data preprocessing rather than metric helper functions.

Reply:
We tried to move it to preprocessing, however we get circular inputs, because the function relies on the metrics package, but is at the same time needed for some metrics.